youtube.css

```
@@ -0,0 +1,155 @@
+html {
+  border: 2px solid gray;
+  margin-left: auto;
+  margin-right: auto;
+  width: 1500px;
+}
+
+body {
+  background-color: whitesmoke;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: repeat(10, auto);
+}
+
+aside {
+  grid-column: 3 / span 1;
```

`span 1` is the default.
```suggestion
  grid-column-start: 3;
```